docs/Bed_Mesh.md

```diff
@@ -372,14 +372,19 @@ are identified in green.
 
 ### Adaptive Meshes
 
-Adaptive bed meshing is a way to speed up the bed mesh generation by only probing
-the area of the bed used by the objects being printed. When used, the method will
-automatically adjust the mesh parameters based on the area occupied by the defined
-print objects.
+#### Concept:
+
+Adaptive bed meshing is a way to speed up the bed meshing process as well as waste less mesh information by only probing
+the area of the bed that will be used for printing. When used, this method will
+automatically adjust the mesh parameters based on the area occupied by objects defined by the `EXCLUDE_OBJECT` command. See the [exclude objects guide](Exclude_Object.md) and
```

Comment:
I actually think that the old version ("area occupied by the defined print objects") was a bit better. Talking about how those objects get defined (the `EXCLUDE_OBJECTS` command) seem more appropriate for the "How it works" section.